hr_expense_tax_adjust/models/hr_expense.py

```diff
@@ -32,9 +32,12 @@ def _get_expense_price_tax(self):
 
     @api.depends("currency_id", "tax_ids", "total_amount", "unit_amount", "quantity")
     def _compute_amount_price_tax(self):
-        for expense in self.filtered(lambda l: not l.tax_adjust):
-            price_tax = expense._get_expense_price_tax()
-            expense.price_tax = price_tax
+        for expense in self:
```

Comment:
Use `continue` for system don't check 2 condition.

```suggestion
        for expense in self:
            if not expense.tax_ids:
                expense.price_tax = 0.0
                continue

            if not expense.tax_adjust:
                expense.price_tax = expense._get_expense_price_tax()
```

Reply:
fixed